test/Microsoft.AspNet.Mvc.Razor.Test/RazorViewEngineTest.cs

```diff
@@ -73,50 +73,82 @@ public static IEnumerable<object[]> ViewLocationExpanderTestData
         [Theory]
         [InlineData(null)]
         [InlineData("")]
-        public void FindView_ThrowsIfViewNameIsNullOrEmpty(string viewName)
+        public void FindView_IsMainPage_ThrowsIfViewNameIsNullOrEmpty(string viewName)
```

Comment:
Reduced this somewhat but still have (just) a few with unique names, similar to old `_IsPartial` name qualifications.